api/src/models/api/nova/chronicle/IAddressBalanceResponse.ts

```diff
@@ -0,0 +1,18 @@
+import { IResponse } from "../../IResponse";
+
+export interface IAddressBalanceResponse extends IResponse {
+    /**
+     * The total balance (including Expiration, Timelock and StorageDepositReturn outputs)
+     */
+    totalBalance?: number;
+
+    /**
+     * The balance of trivialy unlockable outputs with address unlock condition.
```

Comment:
This comment should be updated, its not trivialy unlockable anymore

Reply:
How should it read now ? 😅

Reply:
The balance of all spendable outputs by the address at this time?